mlflow/tracking/fluent.py

```diff
@@ -636,7 +637,7 @@ def log_param(key: str, value: Any, synchronous: bool = True) -> Any:
             values up to length 6000, but some may support larger values.
         synchronous: *Experimental* If True, blocks until the parameter is logged successfully. If
             False, logs the parameter asynchronously and returns a future representing the logging
-            operation.
+            operation. If None, read from environment variable `MLFLOW_ENABLE_ASYNC_LOGGING`.
```

Comment:
What if the environment variable isn't set? We should clarify that this defaults to `False` if the env var isn't set

Reply:
good catch, makes sense to call out in the API!